spec/javascripts/components/copy-to-clipboard-spec.js

```diff
@@ -25,7 +25,8 @@ describe('Copy to clipboard component', function () {
 
     loadCopyToClipboard()
 
-    $('.govuk-button').trigger('click')
+    var copyButton = document.querySelector('button')
+    copyButton.dispatchEvent(new window.Event('click'))
```

Comment:
Is it worth using the `GOVUK.triggerEvent` script here? Might make it more cross browser compatible.

(there's a few other places like this as well)

Reply:
Have updated to use the new `triggerEvent` utility